server/planning/feed_parsers/superdesk_planning_xml.py

```diff
@@ -0,0 +1,194 @@
+            self.parse_content_meta(tree, item)
+            self.parse_news_coverage_status(tree, item)
+            self.parse_news_coverage_set(tree, item)
+            return [item]
+
+        except Exception as ex:
+            raise ParserError.parseMessageError(ex, provider)
+
+    def parse_item_meta(self, tree, item):
+        """Parse itemMeta tag
+
+        :param tree: tree
+        :param item: planning item
+        """
+        meta = tree.find(self.qname("itemMeta"))
+
+        versioncreated_elt = meta.find(self.qname("versionCreated"))
+        editor_note = meta.find(self.qname("edNote"))
+        item_class_elt = meta.find(self.qname("itemClass"))
+        if versioncreated_elt is not None and versioncreated_elt.text:
+            item["versioncreated"] = self.datetime(meta.find(self.qname("versionCreated")).text)
+        if editor_note is not None and editor_note.text:
+            item["ednote"] = editor_note.text
+        if item_class_elt is not None:
+            item["item_class"] = item_class_elt.get("qcode")
+
+    def parse_content_meta(self, tree, item):
+        """Parse contentMeta tag
+
+        :param tree: tree
+        :param item: planning item
+        """
+        content_meta = tree.find(self.qname("contentMeta"))
+        # subjects
+        self.parse_content_subject(content_meta, item)
+
+        firstcreated_elt = content_meta.find(self.qname("contentCreated"))
+        if firstcreated_elt is not None and firstcreated_elt.text:
+            item["firstcreated"] = self.datetime(firstcreated_elt.text)
+
+        updated_elt = content_meta.find(self.qname("contentModified"))
+        if updated_elt is not None and updated_elt.text:
+            item["_updated"] = self.datetime(updated_elt.text)
+
+        slugline_elt = content_meta.find(self.qname("headline"))
+        if slugline_elt is not None and slugline_elt.text:
+            item["slugline"] = slugline_elt.text
+            item["name"] = slugline_elt.text
+
+        description_elt = content_meta.find(self.qname("description"))
+        if description_elt is not None and description_elt.text:
+            item["description_text"] = description_elt.text
+
+        # Assigning the planning date from subject
+        subject_elt = content_meta.find(self.qname("subject"))
+        if subject_elt is not None:
+            assigne_elt = subject_elt.find(self.qname("related"))
+            if assigne_elt is not None:
+                item["planning_date"] = self.datetime(assigne_elt.get("value"))
+        else:
+            item["planning_date"] = utc_to_local(config.DEFAULT_TIMEZONE, utcnow())
+
+    def parse_news_coverage_status(self, tree, item):
+        """Parse assert tag
+
+        :param tree: tree
+        :param item: planning item
+        """
+        assert_el = tree.find(self.qname("assert"))
+        if assert_el is not None:
+            news_coverage_status = assert_el.find(self.qname("newsCoverageStatus")).get("qcode")
```

Comment:
It might be a good idea to use the CV for this (especially for translations, name changes etc).

Similar to EventsML `eventoccurstatus`
https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/feed_parsers/events_ml.py#L97